.github/workflows/release_weekly.yml

```diff
@@ -4,6 +4,7 @@ on:
   schedule:
     # Every Monday at 14:00
     - cron: "0 14 * * 1"
+  workflow_dispatch:
```

Comment:
Should there be something under this or is an empty section like this fine?

Reply:
An empty section is OK - just looks a bit janky in YAML!